src/main/java/seedu/address/ui/DoctorCard.java

```diff
@@ -64,5 +67,9 @@ public DoctorCard(Doctor doctor, int displayedIndex) {
         doctor.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        doctor.getAppointments().stream()
+                .sorted(Comparator.comparing(Appointment::getAppointmentTime))
+                .forEach(appointment -> appointments.getChildren()
+                        .add(new Label(appointment.getAppointmentTime().toString())));
```

Comment:
I think we should update the label to show both the appointment time and the corresponding patient. Similarly for the patient card.